src/mg/PAMI/Client/Impl/ClientImpl.php

```diff
@@ -282,7 +282,11 @@ public function process()
                 $bMsg .= 'ActionId: ' . $this->_lastActionId . "\r\n" . $aMsg;
                 $event = $this->_messageToEvent($bMsg);
                 $response = $this->findResponse($event);
-                $response->addEvent($event);
+                if ($response === false) {
```

Comment:
Exactly what I mentioned earlier, which basically reduces to how to reproduce this behavior, including the asterisk version, action(s) sent (if any), and events/responses that produce this behavior.